wiki/History_of_osu!/2008/en.md

```diff
@@ -36,41 +36,41 @@ A major milestone in osu! history was reached with the addition of the [Multi (m
+The editor, multiplayer, storyboarding, graphics, and chat console all received numerous improvements this month. The Options screen received a new layout, and [custom key bindings](/wiki/Options/Keyboard_Bindings) were introduced.
 
 ## August
 
-Continuing the trend set by July, one would be hard pressed to find an aspect of osu! that wasn't improved or optimised in this month. In the realm of brand new additions, tags were added to beatmaps and multiple monitor support was implemented on an experimental level. A new method of unpausing gameplay was constructed to stop people abusing the pause function (you now need to align your cursor to where it was before you paused in order to continue).
+Continuing the trend set by July, one would be hard pressed to find an aspect of osu! that wasn't improved or optimised in this month. In the realm of brand new additions, tags were added to beatmaps and multiple monitor support was implemented on an experimental level. A new method of unpausing gameplay was constructed to stop people abusing the pause function (which now required players to align their cursor to where it was before they paused in order to continue).
 
-Personal online best scores were added to song selection, fulfilling a long-time request. This required some fairly hefty database optimisation. Graphical improvements were made to the ranking screen, and various elements of the default skin, in an aim to make it feel more clean and crisp than before. The display of slider tick score sprites (10/30) was implemented. Improvements were seen in the multiplayer match setup, allowing host control to be transferred. Many new chat commands were added for user convenience.
+Personal online best scores were added to song selection, fulfilling a long-time request, which required some fairly hefty database optimisation. Graphical improvements to the ranking screen and various elements of the default skin were made in an aim to make them feel even more clean and crisp than before. The display of slider tick score sprites (10/30) was implemented. Improvements were seen in the multiplayer match setup, allowing host control to be transferred. Many new [chat commands](/Chat_Console#commands-list) were added for user convenience.
 
-A new ranked status "![](img/fire.gif) [approved](/wiki/Approved)" was added for maps which were of great quality but exceed ranking length or difficulty (or other) limits in some way. Maps in this status don't add to the player's scores, but have scoreboards and act otherwise like a ![](img/heart.gif) [ranked](/wiki/Ranked) map.
+A new ranked status "![](img/fire.gif) [Approved](/wiki/Approved)" was added for maps which were of great quality but exceed ranking length, difficulty, and/or other limits in some way. Maps in this status don't add to the player's scores, but have scoreboards and act otherwise like a ![](img/heart.gif) [Ranked](/wiki/Ranked) map.
```

Comment:
```suggestion
A new ranked status "![](img/fire.gif) [Approved](/wiki/Approved)" was added for maps which were of great quality but exceed ranking length, difficulty, or other limits in some way. Maps in this status don't add to the player's scores, but have scoreboards and act otherwise like a ![](img/heart.gif) [Ranked](/wiki/Ranked) map.
```